src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+    private static final char OPERATION_ADD = '+';
+    private static final char OPERATION_SUB = '-';
+    private static final char OPERATION_DIV = '/';
+    private static final char OPERATION_MUL = '*';
+    private static final char OPERATION_POW = '^';
+    private static Calculator calculator;
+    
+    @BeforeAll
+    public static void beforeAll() {
+        calculator = new Calculator();
+    }
+    
+    @Test
+    public void calculate_addition_OK() {
```

Comment:
```suggestion
    public void calculate_addition() {
```
try to create uniform names for test methods